### DISCUSSION TITLE
Welcome to RTL Discussions!

### DISCUSSION THREAD

**Discussion by saubyk** [Ride-The-Lightning/RTL#768] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Provide feedback
  * Engage with other community members
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us what you like about RTL and what you'd like to be improved.

**Comment by dlp2050** [comment]
GM All
   I am new to running nodes and lighting network, I am on start9 embassy ride the lighting, I would like to rebalance but the buttons for circular rebalance and loop out is not present, Im sure its just a setting somewhere that I had missed, May I get a little help please?
![tempImagevnYM6i](https://github.com/Ride-The-Lightning/RTL/assets/136254593/d668a49f-6760-4cad-bd6a-51e1ae685d6f)

**Comment by ShahanaFarooqui** [comment]
Loop is only available after configuring but channel rebalancing is available by default. Which LND and RTL version are you on?

**Comment by dlp2050** [comment]
Thank you for your quick response I am not using LND I'm using core lighting 23.02.2~3 and RTL 0.13.6~1

**Comment by ShahanaFarooqui** [comment]
Channel rebalance and loop both are not available for CLN nodes in RTL.

**Comment by dlp2050** [comment]
Thank you so much Shahana, that answers lots do you have any Suggestions?
